### PR TITLE
Pagination Component

### PR DESCRIPTION
**Pull request by porleaaron**
## Description

Please describe the aim of the pull request and the changes made in the commits.

## Purpose

- [x] Pagination Component
- [ ] New feature
- [ ] Other

## How was this tested? (if applicable)

Please describe how the proposed changes have been tested, and the outcome of the tests.

## Test Platform Details (if applicable)

**Operating system:** OSX/Windows 10/Ubuntu 20.04/etc.

**CLI version:**

**Hypervisors/Platforms (if applicable):** qemu/hyper-v/google cloud platform/vmware workstation/etc

**Kernel version (if applicable):**

## Checklist

- [x] Code is commented
- [ ] Unit test coverage encompasses new code
- [ ] Existing unit tests pass with these changes
- [ ] PR is signed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@emotion/react": "^11.9.3",
     "@emotion/styled": "^11.9.3",
+    "@heroicons/react": "^2.0.16",
```
Comment (stefan-kracht) [inline package.json:8]: please uninstall :)

**Review by stefan-kracht — CHANGES_REQUESTED** [review]
See the subissues I added to DIR-91

File: src/design/Pagination/index.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  align?: "center" | "left" | "right";
+}
+
+export const Pagination: React.FC<PaginationProps> = ({
+  children,
+  align = "right",
+}) => (
+  <div
+    className={clsx(
+      "flex border-t border-gray-2 bg-white px-4 py-3 sm:px-6 ",
```
Comment (stefan-kracht) [inline src/design/Pagination/index.tsx:15]: Dark mode colors are missing here. Please check the storybook in dark mode too, it's very easy to see that the Pagination stands on a white background in darkmode.

File: .eslintrc.js [hunk 3]
```diff
@@ -109,6 +109,7 @@ module.exports = {
     "react/prop-types": "off",
     // this is the default in typescript and we want to enforce it in JavaScript as well
     "prefer-const": "error",
+    "no-nested-ternary": "off"
```
Comment (stefan-kracht) [inline .eslintrc.js:112]: This is very important, please never deactivate an eslintrule without communication. I'm happy to discuss any rule that is listed here.

**Review by stefan-kracht — CHANGES_REQUESTED** [review]

File: .eslintrc.js (hunk 3 above)
Comment (porleaaron) [reply .eslintrc.js:112]: Yeah, I will make it sure

File: src/design/Pagination/index.tsx [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    "relative inline-flex items-center px-4 py-2 text-sm font-semibold text-gray-900 ring-1 ring-inset ring-gray-3 hover:bg-gray-1 focus:z-20 focus:outline-offset-0 cursor-pointer dark:bg-gray-12 dark:text-gray-1";
+  const activeClass =
+    "relative z-10 inline-flex items-center bg-gray-12 px-4 py-2 text-sm font-semibold text-gray-1 focus:z-20 focus-visible:outline focus-visible:outline-2 focus-visible:outline-offset-2 focus-visible:outline-primary-600 cursor-pointer dark:bg-gray-dark-12 dark:text-gray-dark-1";
+
+  return icon ? (
+    <button
+      ref={ref}
+      key={key}
+      onClick={onClick}
+      className="relative inline-flex cursor-pointer items-center p-2 text-gray-4 ring-1 ring-inset ring-gray-3 hover:bg-gray-1 focus:z-20 focus:outline-offset-0 dark:bg-gray-12 dark:text-gray-1"
```
[the remark this answers is not in the document]
Comment (porleaaron) [reply src/design/Pagination/index.tsx:50]: Yeah, I rechecked it

File: src/design/Pagination/index.tsx (hunk 2 above)
Comment (porleaaron) [reply src/design/Pagination/index.tsx:15]: yes, I checked all again

**Pull request merged**